### ISSUE TITLE
SDL Example does not build

### ISSUE THREAD

**Issue by jalornales** [Madour/LDtkLoader#32]
```
cmake --build .
[14/17] Building CXX object examples/SDL/CMakeFiles/LDtkSDL.dir/main.cpp.o
FAILED: examples/SDL/CMakeFiles/LDtkSDL.dir/main.cpp.o 
/usr/bin/clang++  -I../examples/SDL/../../include -I../include -isystem /usr/include/SDL2  -MD -MT examples/SDL/CMakeFiles/LDtkSDL.dir/main.cpp.o -MF examples/SDL/CMakeFiles/LDtkSDL.dir/main.cpp.o.d -o examples/SDL/CMakeFiles/LDtkSDL.dir/main.cpp.o -c ../examples/SDL/main.cpp
../examples/SDL/main.cpp:10:17: error: no matching constructor for initialization of 'ldtk::World'
    ldtk::World world;
                ^
../examples/SDL/../../include/LDtkLoader/World.hpp:22:9: note: candidate constructor not viable: requires 1 argument, but 0 were provided
        World(const World&) = delete;
        ^
../examples/SDL/../../include/LDtkLoader/World.hpp:23:9: note: candidate constructor not viable: requires 1 argument, but 0 were provided
        World(World&&) = default;
        ^
../examples/SDL/../../include/LDtkLoader/World.hpp:56:9: note: candidate constructor not viable: requires 4 arguments, but 0 were provided
        World(const nlohmann::json& j, Project* p, const FileLoader& file_loader, bool external_levels);
        ^
../examples/SDL/main.cpp:14:15: error: no member named 'loadFromFile' in 'ldtk::World'
        world.loadFromFile("assets/level.ldtk");
        ~~~~~ ^
../examples/SDL/main.cpp:51:18: error: no member named 'position' in 'ldtk::Tile'
            tile.position.x, tile.position.y,
            ~~~~ ^
../examples/SDL/main.cpp:51:35: error: no member named 'position' in 'ldtk::Tile'
            tile.position.x, tile.position.y,
                             ~~~~ ^
../examples/SDL/main.cpp:56:18: error: no member named 'texture_position' in 'ldtk::Tile'
            tile.texture_position.x, tile.texture_position.y,
            ~~~~ ^
../examples/SDL/main.cpp:56:43: error: no member named 'texture_position' in 'ldtk::Tile'
            tile.texture_position.x, tile.texture_position.y,
                                     ~~~~ ^
6 errors generated.
[15/17] Building CXX object CMakeFiles/LDtkLoader.dir/src/World.cpp.o
ninja: build stopped: subcommand failed.
```

**Comment by Madour** [comment]
Thanks for reporting this issue. It seems I didn't update the sdl example for à long time. Sorry fir the inconveniance, I'll fix this evening

**Comment by Madour** [comment]
Fixed with commit 8c6c870
